test/parallel/test-regress-GH-2245.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable strict */
```

Comment:
in strict mode running this will fail as `bar` is undefined until the code has been `eval`ed. If this line is not present the build will fail lint